pkg/apis/pipeline/v1alpha1/workspace_types.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"path/filepath"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	// WorkspaceDir is the root directory used for PipelineResources and (by default) Workspaces
+	WorkspaceDir = "/workspace"
```

Comment:
Could this be a candidate for `pkg/apis/pipeline/constants.go` to remove it from api versioning "alpha1" pkg?  I'm not totally clear on when the right time to put constants outside the versioned packages is though.  @vdemeester wdyt?

Reply:
oh dear, i didnt know `constants.go` existed - tbh I am not a fan of naming files after the implementation details of their contents, id much rather create file (and package names) that reflect the meanings of their contents - looking at `constants.go` maybe a name for that that makes sense would be something like `storagetypes.go`?

i dont mind moving this outside `v1alpha` - id probably move it into something like `pkg/apis/pipeline/workspace.go` tho

Reply:
done!

Reply:
im actually gonna put it in the workspace package...

Reply:
... nope im not, that causes an import cycle

Reply:
also created https://github.com/tektoncd/pipeline/pull/1730 cuz i am bad at letting things go